src/neuroconv/utils/types.py

```diff
@@ -1,12 +1,7 @@
-from pathlib import Path
-from typing import Optional, TypeVar, Union
+from typing import Optional, Union
 
 import numpy as np
 
-FilePathType = TypeVar("FilePathType", str, Path)
```

Comment:
Yes, this is a hard deprecation

I could not think of a good way to do a soft one without utilizing module-level `__getattr__` magic to inject a deprecation warning